slices/libgcc-s1.yaml

```diff
@@ -4,5 +4,12 @@ slices:
   libs:
     essential:
       - libc6_libs
+      - libgcc-s1_copyright
```

Comment:
```suggestion
```

This should be a global dependency for all slices in this pkg. Ie. after `package: libgcc-s1`, we should have

```yaml
essential:
  - libgcc-s1_copyright
```